src/App.vue

```diff
@@ -137,30 +137,69 @@ function filterLocalImgElements(
 }
 async function updateImageUrls(localImgElements: HTMLImageElement[]) {
   for (const el of localImgElements) {
-    const src = el.src.split('/').pop() as string
-    const blobUrl = await parseImageUrl(src)
+    // trim 'mediaBasePath'
+    // remove leading '.' and '/'
+    const srcPath = el.src
+      .replace(mediaBasePath, '')
+      .replace(/$(\.|\/)/, '')
+      .replace(/$(\.|\/)/, '')
```

Comment:
```suggestion
      .replace(/$(\.\/)/, '')
      .replace(/$(\/)/, '')
```

Reply:
or `replace(/\//, '').replace(/\./,'')`
```suggestion
      .replace(/$(\.|\/)/, '')
      .replace(/$(\.|\/)/, '')
```

Reply:
I don't think `$` is needed

Reply:
>or replace(/\//, '').replace(/\./,'')

we must not remove `.` only. so it has to be either `./` or `/`

Reply:
>I don't think $ is needed

we must be strict to check that if filename starts with `./` or `/`. Without `$`, `replace` will replace the first occurance.
If we have `car.png` then it will replace `.` and the name will become `carpng`

Reply:
`$` should be `^`. will be fixed in upcoming PR